docs/en_US/Release_v1.0.md

```diff
@@ -0,0 +1,33 @@
+
+<p align="center">
+<img src=".././img/release-1-title-1.png" width="100%" />
+</p>
+
+
+From September, 2018 to September, 2019, We are still moving on …
+
+
+ **Great news!**&nbsp;&nbsp;With the tag of **Scalability** and **Ease of Use** , NNI v1.0 is comming. Based on the various types of [Tuning Algorithms](./Tuner/BuiltinTuner.md),NNI has supported the [Neural Architecture Search](docs/en_US/AdvancedFeature/GeneralNasInterfaces.md),which is an exciting news for algorithmic engineers, besides these, NNI v1.0 has made many improvements in the [platform diversification](./TrainingService/SupportTrainingService.md) and [WebUI simplification](./Tutorial/WebUI.md).NNI has grown into a more intelligent automated machine learning(AutoML) toolkit.
+ 
+
+
+<br/>
+<br/>
+<br/>
+<p align="center">
+<img src=".././img/nni-1.png" width="80%" />
+</p>
+
+<br />
+<br />
+
+<p align="center">
+<img src=".././img/release-1-title-2.png" width="100%" />
+</p>
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**Step one**：Start with the [Tutorial Document](./Tutorial/Installation.md),and install NNI v1.0 first.<br>
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**Step two**：Find a " Hello word  example", follow the [Tutorial Documentation](./Tutorial/QuickStart.md) and have a Quick Start. <br>
```

Comment:
Hollo Word? Or Mnist?

Reply:
maybe change to "Hello word(Mnist) example"  ?